templates/org/home.tmpl

```diff
@@ -5,6 +5,7 @@
 		<div id="org-info">
 			<div class="ui header">
 				{{.Org.DisplayName}}
+				<a href="{{.Org.DisplayName}}.rss"><i id="rss-feed" class="ui grey icon tooltip" data-contents="{{.i18n.Tr "rss_feed"}}" data-position="top center">{{svg "octicon-rss" 36}}</i></a>
```

Comment:
`{{.Org.DisplayName}}` is incorrect, it can be something like "my good project". You need something like `RepoLink` here, and it can be discovered in source code.

`id="rss-feed"` should be changed to a CSS class

And, `data-contents` is incorrect.